net/src/client.rs

```diff
@@ -0,0 +1,50 @@
+use bufstream::BufStream;
+use crypt::{maple_crypt, MapleAES};
+use packet::Packet;
+use std::{
+    io::{Result, Write},
+    net::TcpStream,
+};
+
+/// A container for various pieces of information pertaining to a Session's
+/// client.
+pub struct MapleClient {
+    pub stream: BufStream<TcpStream>,
+    pub recv_crypt: MapleAES,
+    pub send_crypt: MapleAES,
+}
+
+impl MapleClient {
+    pub fn new(stream: BufStream<TcpStream>, recv_iv: &Vec<u8>, send_iv: &Vec<u8>) -> Self {
+        let recv_crypt = MapleAES::new(recv_iv, 83);
+        let send_crypt = MapleAES::new(send_iv, 83);
```

Comment:
Need to create a constant for the version. Later on we can have this be injected via a config file...